Assets/MixedRealityToolkit/Interfaces/InputSystem/IPointerPreferences.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Provides interface for getting and setting behaviors and
+    /// possible other settings for pointers in the input system.
+    /// Behaviors are described based on pointer type and input type,
+    /// not per pointer. This is to ensure that new pointers that appear
+    /// maintain consistent behavior.
+    /// </summary>
+    public interface IPointerPreferences
+    {
+        /// <summary>
+        /// Gets the <seealso cref="MixedReality.Toolkit.Input.PointerBehavior"/> for a given pointer
+        /// </summary>
+        PointerBehavior GetPointerBehavior(IMixedRealityPointer pointer);
+
+        /// <summary>
+        /// Gets the <seealso cref="PointerBehavior"/> for a given pointer type,
+        /// handedness, and input type
+        /// </summary>
+        /// <typeparam name="T">All pointers that are of this type, or a subclass of this type, will have the given behavior</typeparam>
+        /// <param name="handedness">Specify Handedness.Any to apply to all handedness, or specify a specific handedness to just disable, right, left.</param>
```

Comment:
>Any [](start = 56, length = 3)

Just out of curiousity, does Both also work? I know that this enum has been confusing in the past with some other APIs.

Reply:
Yes it does.